typescript/cli/src/utils/chains.ts

```diff
@@ -99,3 +100,29 @@ export async function detectAndConfirmOrPrompt(
   } catch (e) {}
   return input({ message: `${prompt} ${label}:`, default: detectedValue });
 }
+
+const INFO_COMMAND: string = 'i';
+const DOCS_NOTICE: string =
+  'For more information, please visit https://docs.hyperlane.xyz.';
+
+export async function inputWithInfo({
```

Comment:
Not chain-specific, recommend moving to a more suitable utils file

Reply:
ah yep was following the precedent set by `detectAndConfirmOrPrompt`. Moved both to an `input.ts` now